src/directors/1.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
May be it's better to join all directors into one file?

Reply:
Done